.bazelci/presubmit.yml

```diff
@@ -3,12 +3,12 @@ validate_config: 1
 tasks:
   ubuntu1804:
     environment:
-      GO_HOME: "$HOME/go-1.19.5"
+      GO_HOME: "$HOME/go-1.20.5"
```

Comment:
Can you update the [patch to the latest](http://go/go-release-history#go1.20.7) one as well?

Reply:
Done